muffnn/autoencoder/autoencoder.py

```diff
@@ -426,36 +425,100 @@ def __getstate__(self):
 
         # Add fitted attributes if the model has been fitted.
         if self._is_fitted:
-            state['input_layer_size_'] = self.input_layer_size_
```

Comment:
hmm...  i made that comment, but now that i think about it, it's probably more efficient to set the values individually than make a separate dict, but this is fine.  in practice, it won't matter either way here